src/config/Hit.js

```diff
@@ -41,36 +41,38 @@ export function Hit({ hit, insights, view }) {
 
         <div className="uni-Hit-Actions">
           <button
-            title="Like this product"
+            title="Add to cart"
             className="uni-Hit-ActionButton"
             onClick={(event) => {
               event.preventDefault();
               event.stopPropagation();
 
               insights('convertedObjectIDsAfterSearch', {
-                eventName: 'Product Liked',
+                eventName: 'Product Added to Cart',
               });
             }}
           >
-            <LikeIcon />
+            <CartIcon />
           </button>
         </div>
       </a>
     </article>
   );
 }
 
-function LikeIcon(props) {
+function CartIcon(props) {
   return (
     <svg
       viewBox="0 0 24 24"
       width={20}
       height={20}
       stroke="currentColor"
-      strokeWidth={2}
+      stroke-width={2}
```

Comment:
The official JSX key is `strokeWidth`. Was it not working with our Preact version?

Reply:
No this is a copy/paste from SVG code and I forgot to rename. On it.